### ISSUE TITLE
Line break characters are shown incorrectly

### ISSUE THREAD

**Issue by ask9** [dbeaver/dbeaver#35500]
### Description

use code:

```
select 'hello'::timestamptz;

```

Please also check that the message copied to notepad will have the line break symbol, not the incorrect symbol.

https://github.com/user-attachments/assets/70720d80-92ec-4748-870e-b6d1693d65f0

### DBeaver Version

Community Edition 24.1.5-202408122322

### Operating System

Windows 10

### Database and driver

PostgreSQL 16
PostgreSQL JDBC Driver 42.7.4

**Comment by ask9** [comment]
By the way, perhaps the message here should not show the symbol ¶ as well.
Perhaps it would look better (more easily readable by the user) if it is replaced with a simple space-symbol.

![изображение](https://github.com/user-attachments/assets/b2625ad7-a932-4191-856c-053da60dfc7f)

**Comment by ask9** [comment]
The same issue is here:
- use code:
```
select 
1,
2,
3,
4,
5;
```
https://github.com/user-attachments/assets/2cc81986-6bc2-4388-9a57-3d76855ab327

**Comment by MashaKorax** [comment]
Thanks for your request.

**Comment by Destrolaric** [comment]
![dbeaver_OFsztpjHiF](https://github.com/user-attachments/assets/b6582a68-4c23-4510-a926-47b99cb944d3)
![dbeaver_svFuZx78Fl](https://github.com/user-attachments/assets/064c06b5-e4e4-4541-9788-41a4a36c0f99)
Hi, we have a way to show line breaks inside of the SQL Editor. You can find them in the Text Editors tab, `show whitespace characters`.
Is this a problem mentioned in this issue?

**Comment by ask9** [comment]
@Destrolaric  no, it is not related to the current issue. Your option is only applicable to sql editor what is very useful when editing text.
But showing these symbols in error messages, sql queries in Query Manager and names of a tab - only leads to additional strain on eyes, decrease readability and doesn't bring some value for a user for my point of view (and that's exactly what this issue is about).
That's why I suggest to replace them with space-character always, in spite of any options in Preferences.

**Comment by ask9** [comment]
I forgot to mention another one location: text in Query Manager.
Maybe you will be against it, but I suggest you also to consider replacing this character in Query Manager as well.
It would enhance readability and it is shown in this example:
- execute code
```
select 
1,
2,
3,
4,
5;

select 1, 2, 3, 4, 5;
```

![image](https://github.com/user-attachments/assets/578ce700-e885-4499-9d43-f3fdcbd9a471)
